lib/simplifyExpression/functionsSearch/nthRoot.js

```diff
@@ -463,11 +479,10 @@ function isMultiplicationOfEqualNodes(node) {
     return false;
   }
 
-  // return if they are all equal nodes
-  return node.args.reduce((a, b) => {
-    return a.equals(b);
-  });
+  const terms = node.args.map(print);
 
+  // return if they are all equal nodes
+  return [...new Set(terms)].length === 1;
```

Comment:
Neat! I'm wondering if it's less readable - probably okay but Lili can maybe confirm